core/config/initializers/00_tenant_config_schema.rb

```diff
@@ -29,6 +29,16 @@
           description: "Name displayed in the browser title bar and in the public pages",
           default: "My Webstore"
         },
+        organization_requirements: {
+          type: "array",
+          title: "Webstore organization requirements",
+          description: "Information required for new organizations",
+          items: {
+            type: "string",
+            enum: MnoEnterprise::Organization::REQUIRABLE_FIELDS
```

Comment:
Change the constant to be
```ruby
REQUIRABLE_FIELDS = ['main_address']
```
Then you can have:
```ruby
enum: MnoEnterprise::Organization::REQUIRABLE_FIELDS,
'x-schema-form': {
  titleMap: Hash[MnoEnterprise::Organization::REQUIRABLE_FIELDS.map{|f| [f, f.titleize]}]
}
```

so you'll have `main_address` saved in the backend (no longer need to manipulate the string) and `Main Address` in the UI.